Dockerfile

```diff
@@ -9,6 +9,7 @@ RUN tar -xzf /cachi2/output/deps/generic/check-payload-${CHECK_PAYLOAD_VERSION}.
     CGO_ENABLED=0 go build -ldflags="-X main.Commit=${CHECK_PAYLOAD_VERSION}" -o /opt/app-root/src/check-payload-binary && \
     chmod +x /opt/app-root/src/check-payload-binary
 
+
```

Comment:
why this empty line is for ?

Reply:
that was accidental. removed it. thanks for catching that